packages/react-core/src/components/Tooltip/examples/TooltipOptions.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { Button, Tooltip, Checkbox, Select, SelectOption, TextInput, TooltipPosition } from '@patternfly/react-core';
+
+export const TooltipOptions: React.FunctionComponent = () => {
+  const [trigger, setTrigger] = React.useState(['mouseenter', 'focus']);
+  const [isVisible, setIsVisible] = React.useState(true);
+  const [contentLeftAligned, setContentLeftAligned] = React.useState(false);
+  const [enableFlip, setEnableFlip] = React.useState(true);
+  const [position, setPosition] = React.useState<TooltipPosition>(TooltipPosition.top);
+  const [positionSelectOpen, setPositionSelectOpen] = React.useState(false);
+  const [flipSelectOpen, setFlipSelectOpen] = React.useState(false);
+  const [flipBehavior, setFlipBehavior] = React.useState('flip');
+  const [entryDelayInput, setEntryDelayInput] = React.useState(0);
+  const [exitDelayInput, setExitDelayInput] = React.useState(0);
+  const [animationDuration, setAnimationDuration] = React.useState(300);
+  const tipBoxRef = React.useRef(null);
+
+  const scrollToRef = (ref: React.RefObject<any>) => {
+    if (ref && ref.current) {
+      ref.current.scrollTop = 400;
+      ref.current.scrollLeft = 300;
+    }
+  };
+
+  React.useEffect(() => {
+    scrollToRef(tipBoxRef);
+  }, []);
+
+  return (
+    <>
+      <div>
+        <div style={{ border: '1px solid' }}>
+          <Checkbox
+            label="trigger: mouseenter"
+            isChecked={trigger.includes('mouseenter')}
+            onChange={checked => {
+              let updatedTrigger: string[] = [];
+              checked && (updatedTrigger = trigger.concat('mouseenter'));
+              !checked && (updatedTrigger = trigger.filter(t => t !== 'mouseenter'));
+              setIsVisible(false);
+              setTrigger(updatedTrigger);
+            }}
+            aria-label="trigger: mouseenter"
+            id="trigger_mouseenter"
+          />
+          <Checkbox
+            label="trigger: focus"
+            isChecked={trigger.includes('focus')}
+            onChange={checked => {
+              let updatedTrigger;
```

Comment:
I don't have a conviction that using `let` in cases like these is bad practice. 

I wonder if you could shorten the code block to something like
```
const updatedTrigger = checked ? trigger.concat('focus) : trigger.filter(t => t !== 'focus');
```